edge-modules/TwinTester/Program.cs

```diff
@@ -18,35 +18,61 @@ static async Task Main()
         {
             Logger.LogInformation($"Starting twin tester with the following settings:\r\n{Settings.Current}");
 
+            Console.WriteLine(StatusCode.DesiredPropertyReceived.ToString());
```

Comment:
```suggestion
            Logger.LogInformation(StatusCode.DesiredPropertyReceived.ToString());
```

Reply:
I removed because it doesn't make sense to print that status

---
In reply to: [359098584](https://github.com/Azure/iotedge/pull/2108#discussion_r359098584) [](ancestors = 359098584)